src/pg/.gitignore

```diff
@@ -0,0 +1,6 @@
+regression.diffs
+regression.out
+results/
+crankshaft--dev.sql
+crankshaft--dev--current.sql
+crankshaft--current--dev.sql
```

Comment:
also I have my reservations about committing the virtualenv here. It shall only contain the crankshaft package, right?